tests/test_autodiscover.py

```diff
@@ -36,7 +36,7 @@ def test_autodiscover_with_components_as_views(self):
 
         all_components_after = component_registry.registry.all().copy()
         imported_components_count = len(all_components_after) - len(all_components_before)
-        self.assertEqual(imported_components_count, 1)
+        self.assertEqual(imported_components_count, 2)
```

Comment:
This is a very strange test. Checks that we didn't add any new autodiscoverable components?

Reply:
Hm, looking at the test, I think the test would make more sense if we checked that `autoimport()` imports the components from the `tests/components` dir.

For now I've put a TODO there. I will be reviewing these tests when I'll be making an MR for autoimport, as mentioned here https://github.com/EmilStenstrom/django-components/issues/527#issuecomment-2183091056